src/main/java/seedu/java/util/Task.java

```diff
@@ -0,0 +1,47 @@
+package seedu.java.util;
+
+public class Task {
```

Comment:
Perhaps making Task an abstract class can be considered, since a Task is never created. Todo, Event, and Deadline can simply extend it. Methods such as Task#completeTask, Task#getTask can be kept, while perhaps abstract methods can be implemented.